### ISSUE TITLE
[Content] Review direct deposit content page and suggest updates

### ISSUE THREAD

**Issue by Samara-Strauss** [department-of-veterans-affairs/va.gov-team#2903]
We want to add some additional documentation to the direct deposit content page, specifically making it clearer how to switch between paper checks <> direct deposit.

Also, the page could probably use one more once-over to make sure there are no other changes we want to make.

### Definition of done

- [x] Samara to conduct design review
- [x] Discuss ideas with Peggy
- [x] Peggy to work on updates

**Comment by Samara-Strauss** [comment]
@peggygannon for awareness, I am going to review the direct deposit content page. There are a couple updates I want to make. Just wanted to let you know that this will be coming down the pipeline at some point. 

I know you have a million other things to do at any given point, so let's prioritize accordingly. This is not an immediate/this week need. Do you think it could be put in your pipeline for next sprint?

**Comment by peggygannon** [comment]
Hi, @Samara-Strauss thank you for the update. 

Yes, I can slot this in on my next sprint. Please let me know when you start to have discussions around this update.

**Comment by Samara-Strauss** [comment]
cc/ @lkoenigsberg @Justin-Pickett for awareness that I created this ticket.

**Comment by Samara-Strauss** [comment]
@peggygannon I spent some time this afternoon thinking through potential updates to the content. Updates I want to make are:

1. Clarifying the narrative a bit if people scan headings and don't read every word.
2. Making it easy to find information on the switch between paper checks <> direct deposit, and clearly linking to each form.

I'd also like to revisit the whole last section about updating other information changes.

Let me know if you have time to talk about this before next Wednesday, 11/6. Actual execution of this work can wait until next sprint, but I'll be OOO from 11/6 – 11/11 so I wanted to make sure you were set up to start things whenever you were ready.

And no worries if you are swamped right now — we can wait until I return to regroup if that's what works best for you. I'll be back 11/12. Just let me know either way and I'll get something on the calendar.

**Comment by peggygannon** [comment]
Hi, @Samara-Strauss I have some time next Tuesday, btw. 10:30 a.m.-noon PT or 1 p.m.-2 p.m. PT. Does anytime in there work?

**Comment by Samara-Strauss** [comment]
@peggygannon let's do Tuesday at 1:30 ET/10:30 PT. I'll put something on the calendar.

**Comment by Samara-Strauss** [comment]
Invite sent! Let me know if you don't see it for some reason.

**Comment by Samara-Strauss** [comment]
Hey @peggygannon — Below are screenshots of the ideas we discussed for updating the direct deposit content page. To recap:

1. I want the page to be easier to scan by heading. One update I made in this regard is moving the intro about updating direct deposit in your profile for comp & pen/updating on eBenefits for education to below each benefit heading instead of below "How can I change my VA direct deposit information online?"
2. I also modified some of the headings in the dynamic CTAs to be a little bit more descriptive for people who are going to scan and not read the bulk of the content.
3. I updated the "Are there other ways to change my VA direct deposit or contact information?" section to include more information on different forms people might need and different ways to get their payment info updated. [That is further elaborated on here](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/direct-deposit/discovery-research/paper-checks-documentation.md#what-we-should-communicate-on-vagov).
4. We talked about removing everything under "Should I contact VA if I have other changes to my information?" I think this makes sense, and you're probably right that this was leftover from when the page was more generic.

**Full page with sign in CTA**

[Direct deposit content page: Suggested Fall 2019 updates.pdf](https://github.com/department-of-veterans-affairs/va.gov-team/files/3811591/Direct.deposit.content.page.Suggested.Fall.2019.updates.pdf)

**Verify your identity CTA**

[Direct deposit content page: Verify your identity prompt .pdf](https://github.com/department-of-veterans-affairs/va.gov-team/files/3811594/Direct.deposit.content.page.Verify.your.identity.prompt.pdf)

**2FA CTA**

[Direct deposit content page: Add 2FA prompt.pdf](https://github.com/department-of-veterans-affairs/va.gov-team/files/3811595/Direct.deposit.content.page.Add.2FA.prompt.pdf)

**Comment by Samara-Strauss** [comment]
Also, so I don't forget — noting here that you'll get to this some time during your next sprint, which starts 11/13.

**Comment by peggygannon** [comment]
**11/25/19 update** After syncing with Samara, content updates on on hold pending feedback from eBenefits. I will pick this back up once we have more direction about switching paper checks to direct deposit.

**Comment by Samara-Strauss** [comment]
Adding the blocked label. This will be unblocked as soon as we determine whether we're keeping the new "Add" functionality. We'll be making this determination the week of 12/2.

**Comment by Samara-Strauss** [comment]
@peggygannon — I've got the final 🤞 updates for the direct deposit content page. Let's discuss these soon, along with #3763.

**Full content page**

[More on different ways to update payment info](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/direct-deposit/discovery-research/paper-checks-documentation.md#what-we-should-communicate-on-vagov)

[Direct deposit content page: Nov 2019 pre DS logon gate.pdf](https://github.com/department-of-veterans-affairs/va.gov-team/files/3922652/Direct.deposit.content.page.Nov.2019.pre.DS.logon.gate.pdf)

![Direct deposit content page: Nov 2019 pre DS logon gate](https://user-images.githubusercontent.com/34068740/70156464-f8dc4980-1681-11ea-8413-8d86cfb2a242.png)

**DS Logon gate**

For DS Logon users, they should only ever see the sign in CTA (if they are logged out) and then the DS Logon gate. They should never see any of the other gates.

[Direct deposit content page: DS Logon gate.pdf](https://github.com/department-of-veterans-affairs/va.gov-team/files/3922699/Direct.deposit.content.page.DS.Logon.gate.pdf)

![Direct deposit content page: DS Logon gate](https://user-images.githubusercontent.com/34068740/70156735-67210c00-1682-11ea-8596-044c8f99e28e.png)

**Verify your identity CTA**

This will only ever show to MHV or ID.me users.

[Direct deposit content page: Verify your identity prompt .pdf](https://github.com/department-of-veterans-affairs/va.gov-team/files/3922700/Direct.deposit.content.page.Verify.your.identity.prompt.pdf)

![Direct deposit content page: Verify your identity prompt](https://user-images.githubusercontent.com/34068740/70156757-71dba100-1682-11ea-9148-a349c856466f.png)

**2FA CTA**

This will only ever show to MHV or ID.me users.

[Direct deposit content page: Add 2FA prompt.pdf](https://github.com/department-of-veterans-affairs/va.gov-team/files/3922701/Direct.deposit.content.page.Add.2FA.prompt.pdf)

![Direct deposit content page: Add 2FA prompt](https://user-images.githubusercontent.com/34068740/70156771-756f2800-1682-11ea-87a4-4dad5355ac15.png)

**Comment by Samara-Strauss** [comment]
@peggygannon I am amending the above: **We have made the decision to put the DS Logon gate is on hold for now.** We still need to make 95% of the other updates to this page, so let's keep today's chat to discuss the following: 

**Full content page**

[More on different ways to update payment info](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/direct-deposit/discovery-research/paper-checks-documentation.md#what-we-should-communicate-on-vagov)

[Direct deposit content page: Nov 2019 pre DS logon gate.pdf](https://github.com/department-of-veterans-affairs/va.gov-team/files/3927778/Direct.deposit.content.page.Nov.2019.pre.DS.logon.gate.pdf)

![Direct deposit content page: Nov 2019 pre DS logon gate](https://user-images.githubusercontent.com/34068740/70251314-440f5e80-174d-11ea-8d79-49f0f916da3a.png)

**Verify your identity CTA**

[Direct deposit content page: Verify your identity prompt .pdf](https://github.com/department-of-veterans-affairs/va.gov-team/files/3922700/Direct.deposit.content.page.Verify.your.identity.prompt.pdf)

![Direct deposit content page: Verify your identity prompt](https://user-images.githubusercontent.com/34068740/70156757-71dba100-1682-11ea-9148-a349c856466f.png)

**2FA CTA**

[Direct deposit content page: Add 2FA prompt.pdf](https://github.com/department-of-veterans-affairs/va.gov-team/files/3922701/Direct.deposit.content.page.Add.2FA.prompt.pdf)

![Direct deposit content page: Add 2FA prompt](https://user-images.githubusercontent.com/34068740/70156771-756f2800-1682-11ea-87a4-4dad5355ac15.png)

**Comment by peggygannon** [comment]
Hi, @Samara-Strauss I've made the updates to the Direct Deposit page and have attached a Word doc with screenshots for your review. (I can't get a full-page screenshot out of drupal, so I cobbled them together in Word. Sorry, not the best format!)

The only updates that still need to happen are the copy updates to the React widgets. A developer needs to make them and I have an issue open for Nick to do those today. I'll share the updated screenshots with you when their done. Let me know if you have any questions!

[https://app.zenhub.com/files/133843125/0d626df1-0fd6-4e41-823f-5332bb54fe93/download](https://app.zenhub.com/files/133843125/0d626df1-0fd6-4e41-823f-5332bb54fe93/download)

**Comment by Samara-Strauss** [comment]
Thanks, @peggygannon! Next time, is there any way you could share the copy plain text in a word doc or something? That would make it easier to comment and discuss back and forth. Here are my suggestions.

- Under "Disability compensation and pension benefit payments": In the "Note", can you mention that they'll only need to verify their identities and set up 2FA if they haven't already done this?
- Under "To cancel paper checks and start direct deposit": Can you update "You can cancel paper checks from your VA.gov profile" to something like "You can cancel paper checks from your VA.gov profile and opt-in to receive your compensation payments via direct deposit"?

Otherwise, this looks awesome! Knocked it out of the park. 

> The only updates that still need to happen are the copy updates to the React widgets. A developer needs to make them and I have an issue open for Nick to do those today. 

Any way you can just share the plain text so I can review?

**Comment by peggygannon** [comment]
Hi, @Samara-Strauss thanks for your feedback. I'll update the copy with your recommendations above. Next time I'll put in a word doc for your review. My hope is that someday, i'll just be able to share a preview link out of Drupal. 

Re: the widget updates: 

I talked to Nick today and he said that we can't make copy updates to the widgets at this time, as the widget headers and body copy is also shared by other CTA widgets. The different widget states share the same "service description", so if we change a header on one widget, it will affect other widgets. 

Nick said in the future we can modify this to make them more flexible, but we'd need to plan and scope this work. 

That said, are you OK with moving forward with the widget headers as they appear in the screenshots above?

**Comment by Samara-Strauss** [comment]
Ah, bummer! Thanks for letting me know @peggygannon. I am OK with moving forward with the existing widgets given the way content is shared across pages.

**Comment by peggygannon** [comment]
@Samara-Strauss Ok, sounds good. I'll make the last of the edits above tonight and then we can publish this page with tomorrow's deploy.

**Comment by Samara-Strauss** [comment]
@peggygannon awesome — let me know if anything changes. Otherwise, I will take a look at it after the 3:00 ET deploy!

**Comment by Samara-Strauss** [comment]
It's live and looks great! https://www.va.gov/change-direct-deposit/

Closing!
